clean_project/project/index.py

```diff
@@ -2,27 +2,32 @@
 Create a JSON file used to indexing all words in a config_file.yaml
 """
 import argparse
-import codecs
-import clean_file
+import contextlib
+import urllib
+from ConfigParser import SafeConfigParser
+from collections import defaultdict
+
 import psycopg2
 import yaml
 
-from collections import defaultdict
-from ConfigParser import SafeConfigParser
+import clean_file
 
 # Update index table only if word and file match. Else do nothing
-QUERY_UPDATE = "UPDATE index SET occurrences=%s WHERE word=%s AND file=%s;"
+query_update = "UPDATE index SET occurrences=%s WHERE word=%s AND file=%s;"
 # Insert word and file which exist in index table.
-QUERY_INSERT = ("INSERT INTO index (word, file, occurrences)"
+query_insert = ("INSERT INTO index (word, file, occurrences)"
                 "VALUES (%s, %s, %s);")
 # Select to verify if a key (word, file) exists
-QUERY_SELECT = "SELECT 1 FROM index WHERE word=%s AND file=%s"
+query_select = "SELECT 1 FROM index WHERE word=%s AND file=%s"
```

Comment:
write these 3 variable names in all caps, to mark that they won't change (they're supposed to be constant)